lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -35,150 +33,118 @@
+@teacher{If your students are confident with computing mean, median, and mode, skip to the next section of the lesson, where students will (1) use Pyret to compute various measures of center and (2) begin considering the usefulness of each measure of center.}
 
-But there are many different ways to define "center", and each method has strengths and weaknesses. The shape of the data can play a huge role in whether or not one kind of summary is appropriate!
 
-@slidebreak
+When working with dot plots, we often wondered, "What is a typical value in this sample?"
 
-@lesson-instruction{
-Let's take a moment to consider what values might be typical for the weight of our animals by completing @printable-exercise{typical.adoc}.
-}
+... and we often discovered that there were a variety of correct responses!
 
-@slidebreak 
+When thinking about typicality, there are a few approaches:
 
-@center{@image{images/pounds.png}}
+- identify the value that is most common
+- identify the midpoint of the sample
+- identify a "balance point" among the data points
 
-@QandA{
-@Q{Do you think there is a midpoint of this sample?}
-@A{There are 32 animals - meaning that there is not one point in the middle.}
+It turns out that these different ways of thinking about typicality are all different ways of "measuring the center".  "What value is typical?" is asking "is there a value that all the others tend to cluster around?"
+
+Statisticians have a more formal, mathematically-defined term for this: @vocab{central tendency}. Central tendency is a "summary" measure, that attempts to describe a whole set of data with a single value that represents the middle or "center" of its distribution. In other words, _a value that all the others tend to cluster around_.
 
-@Q{Is there a value that shows up most often in this sample?}
-@A{Since we see that dots are stacked up, it seems likely that there is some repetition in the animals' weights.}
+There are several different measures of central tendency. Data Scientists know what each measure is, and when to use it. And in the next few lessons, you'll learn that too.
 
-@Q{What value did you decide was typical? Why?}
-@A{There isn't one right answer here! The point is for students to hear each other's thinking, recognize that it's hard to summarize the data with a single number, and understand that there are different logical frameworks for doing so.}
+@QandA{
+@Q{Compute the mean, median, and mode for this dataset: 17, 23, 24, 23, 22}
+@A{Mean: 22; median: 23; mode: 23}
+@A{If students struggle to arrive at correct responses, proceed with this lesson as written, at whatever speed and level of depth you deem appropriate for your students. If your students are successful at computing mean, median, and mode, skip to the second section of the lesson. If you know that your students do not know how to compute mean, median, and mode, you can skip this question.}
```

Comment:
If you're encouraging teachers to skip this section based on student responses to this question:

1. Let's give them more than one dataset so that they have to tackle multiple and no modes, median from an even number of data points.
2. Perhaps worth an opt-printable?
3. Perhaps a great use of mdbook quiz so that we know what students are coming into this lesson with!

Reply:
responded to 1 + 2. Resolving this comment now because git issue #2323 exists.